guides/common/modules/con_download-policies-overview.adoc

```diff
@@ -20,7 +20,12 @@ On Demand::
 This setting has no effect if you set a corresponding repository on a {SmartProxy} to *Immediate* because {ProjectServer} is forced to download all the packages.
 
 The *On Demand* policy acts as a _Lazy Synchronization_ feature because they save time synchronizing content.
+ifdef::satellite[]
 The lazy synchronization feature must be used only for `yum` repositories.
+endif::[]
+ifndef::satellite[]
+The lazy synchronization feature must be used only for `deb` and `yum` repositories.
```

Comment:
It may be worth adding that the feature must only be used for additive upstream repos. If the upstream repos is in the habbit of chucking out packages it previously offered, you are going to have a bad time trying to sync it "lazy"...

Reply:
I have added an explanation and used "EPEL" as an example.